src/components/templates/ContentTemplate.tsx

```diff
@@ -52,54 +52,69 @@ const ContentTemplate: FC<Props> = (props) => {
+                      width={106}
+                      height={72}
+                    />
+                    <span
+                      style={{ marginTop: 1, cursor: 'pointer' }}
+                      onClick={() => props.onClickNewSecretCard('ask')}
+                    >
+                      {`${props.newPostCount || 0}개의 비밀카드 도착`}
+                    </span>
+                    <img
+                      onClick={() => props.onClickNewSecretCard('ask')}
+                      src={IMAGES.rightButton}
+                      width={22}
+                      height={22}
+                    />
+                  </NoticeText>
+                  <img
+                    style={{ position: 'relative', bottom: 65, zIndex: -1 }}
+                    src={IMAGES.img_tape_newq}
+                    width={'100%'}
+                  />
+                </>
+              )}
             </NoticeContainer>
             <ContentContainer>
-              {postContent?.ask.map((data, index) => {
-                return (
-                  <QuestionCard
-                    key={index}
-                    id={data.node.id}
-                    questionTitle={data.node.content}
-                    backColor={data.node.color}
-                    stickers={data.node.usedEmoticons}
-                    comments={data.node.comments}
-                    createdAt={data.node.createdAt}
-                    updatedAt={data.node.updatedAt}
-                    secretType={data.node.secretType}
-                    onClickOption={() => {
-                      props.onClickRemove(data.node.id, props.tabIndex)
-                    }}
-                    onClickLike={() => {
-                      props.onClickLike(data.node.id, props.tabIndex)
-                    }}
-                  />
-                )
-              })}
+              {postContent && postContent.ask.length > 0 ? (
+                postContent?.ask.map((data, index) => {
+                  return (
+                    <QuestionCard
+                      key={index}
+                      id={data.node.id}
+                      questionTitle={data.node.content}
+                      backColor={data.node.color}
+                      stickers={data.node.usedEmoticons}
+                      comments={data.node.comments}
+                      createdAt={data.node.createdAt}
+                      updatedAt={data.node.updatedAt}
+                      secretType={data.node.secretType}
+                      onClickOption={() => {
+                        props.onClickRemove(data.node.id, props.tabIndex)
+                      }}
+                      onClickLike={() => {
+                        props.onClickLike(data.node.id, props.tabIndex)
+                      }}
+                    />
+                  )
+                })
+              ) : (
+                <BackgroundSticker src={IMAGES.backgroundSticker} />
+              )}
+              {}
```

Comment:
What’s role??

Reply:
Good Review!

Reply:
I'm using the autocomplete function, so it seems I've made a mistake.